Nightly Testing Report

# Nightly Testing Report
This issue([log](https://github.com/firebase/firebase-ios-sdk/actions/runs/13030864662)) is generated at 01/29/2025 04:03 AM, fetching workflow runs triggered in the last 24 hrs.
| Workflow |01/28|
| -------- | -------- |
| [crashlytics](https://github.com/firebase/firebase-ios-sdk/blob/main/.github/workflows/crashlytics.yml) | [failure](https://github.com/firebase/firebase-ios-sdk/actions/runs/13023439449) |
| [prerelease](https://github.com/firebase/firebase-ios-sdk/blob/main/.github/workflows/prerelease.yml) | [failure](https://github.com/firebase/firebase-ios-sdk/actions/runs/13024962218) |

FIREBASE_CI is not set in prerelease.yml:

Validating spec
 -> FirebaseAuth
 -> FirebaseAuth (11.8.0)
    - NOTE  | xcodebuild:  note: Using codesigning identity override: -
    - NOTE  | xcodebuild:  note: Building targets in dependency order
    - NOTE  | [iOS] xcodebuild:  note: Target dependency graph (15 targets)
    - WARN  | xcodebuild:  FirebaseAuth/FirebaseAuth/Sources/Swift/Backend/RPC/GetOOBConfirmationCodeRequest.swift:180:45: warning: 'dynamicLinkDomain' is deprecated: Firebase Dynamic Links is deprecated. Migrate to use Firebase Hosting link and use `linkDomain` to set a custom domain instead.
    - NOTE  | xcodebuild:  note: Metadata extraction skipped. No AppIntents.framework dependency found. (in target 'FirebaseCoreInternal' from project 'Pods')
    - NOTE  | xcodebuild:  note: Metadata extraction skipped. No AppIntents.framework dependency found. (in target 'FirebaseAuth' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 11.0, but the range of supported deployment target versions is 12.0 to 17.5.99. (in target 'RecaptchaInterop' from project 'Pods')
    - NOTE  | xcodebuild:  note: Using codesigning identity override: 
    - NOTE  | xcodebuild:  note: Target dependency graph (14 targets)

[!] The `/private/tmp/test/firebase-ios-sdk/FirebaseAuth.podspec` specification does not validate.

--- a/.github/workflows/prerelease.yml
+++ b/.github/workflows/prerelease.yml
@@ -9,6 +9,9 @@ on:
     # Run every day at 9pm (PST) - cron uses UTC times
     - cron:  '0 5 * * *'
 
+env:
+  FIREBASE_CI: true
+
 concurrency:
     group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
     cancel-in-progress: true

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -11,6 +11,9 @@ on:
     # Run every day at 9pm (PST) - cron uses UTC times
     - cron:  '0 5 * * *'
 
+env:
+  FIREBASE_CI: true
+
 concurrency:
     group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
     cancel-in-progress: true

